src/pages/RequestCallPage.js

```diff
@@ -206,10 +207,7 @@ class RequestCallPage extends Component {
                     <HeaderWithCloseButton
                         title={this.props.translate('requestCallPage.title')}
                         shouldShowBackButton
-                        onBackButtonPress={() => Report.fetchOrCreateChatReport([
-                            this.props.session.email,
-                            CONST.EMAIL.CONCIERGE,
-                        ], true)}
```

Comment:
Left a similar comment elsewhere, but just want to be certain that we were not depending on a chat to be created at this point. Probably we are not and this is fine.

Reply:
Nope, we weren't depending on it before or now. Before, the call could only be requested from the Concierge DM, so it was guaranteed to already exist. Now we are taking you back to whatever page you came from instead.